src/ctia/flows/crud.clj

```diff
@@ -338,9 +346,10 @@
     fm))
 
 (s/defn ^:private apply-event-hooks :- FlowMap
-  [{:keys [events] :as fm} :- FlowMap]
+  [{{:keys [HooksService]} :services
+    :keys [events] :as fm} :- FlowMap]
   (doseq [event events]
-    (h/apply-event-hooks event))
+    ((:apply-event-hooks HooksService) event))
```

Comment:
Avoided destructuring here because of the risk of infinite recursion (would shadow the current `defn`).